server/server.js

```diff
@@ -142,7 +142,7 @@ server.get('/api/pregnancy-centers/verify', isLoggedInAPI, async (req, res) => {
 	const primaryContact = pregnancyCenter.primaryContact
 
 	if (primaryContact) {
-		const { firstName, lastName, email, phone } = await UserModel.findOne({
+		const user = await UserModel.findOne({
```

Comment:
I realized the response I sent through email didn't show up under the comment! But yeah, it was throwing an error as written, so I had to change it to get it to run, and then realized that it could be more resilient to changes in the User model if it wasn't destructured.